### ISSUE TITLE
Temporary directory "~/" created on build

### ISSUE THREAD

**Issue by uilianries** [conan-io/conan-package-tools#86]
Hi!

Conan package tools 0.4.31 creates a directory "~" in current project directory, when the build starts.

Steps to reproduce:

1) Start a docker container
`$ docker run --rm -ti lasote/conangcc63`

2) Update Conan and Package Tools
`$ sudo pip install -U conan conan_package_tools`

3) Create a empty project
`$ conan new -s -t foobar/0.1.0@qux/baz`

4) Create default build.py
```python
from conan.packager import ConanMultiPackager

if __name__ == "__main__":
    builder = ConanMultiPackager(username="qux")
    builder.add_common_builds()
    builder.run()
```

5) Just run it
`$ python build.py`

As result, conan + conan_package_tools will produce a entry tested package.  
However, a list command will show:
```shelll
conan@e93746325111:~/$ ls
build.py  conanfile.py  src  test_package  ~
```
And finally we have:
`$ file \~/.conan/conan.conf `

I didn't see that behavior before.

**Comment by lasote** [comment]
Reproduced. I'm taking a look.

**Comment by lasote** [comment]
Fixed and released in 0.4.32

**Comment by uilianries** [comment]
Thanks!
